generate_schema/provider/status_changes.json

```diff
@@ -0,0 +1,157 @@
+              "provider_name": {
+                "$id": "#/properties/data/properties/status_changes/items/properties/provider_name",
+                "type": "string",
+                "description": "The public-facing name of the Provider",
+                "examples": [
+                  "Provider Name"
+                ],
+                "pattern": "^(.*)$"
+              },
+              "provider_id": {
+                "$id": "#/properties/data/properties/status_changes/items/properties/provider_id",
+                "description": "The UUID for the Provider, unique within MDS",
+                "$ref": "#/definitions/uuid"
+              },
+              "device_id": {
+                "$id": "#/properties/data/properties/status_changes/items/properties/device_id",
+                "description": "A unique device ID in UUID format",
+                "$ref": "#/definitions/uuid"
+              },
+              "vehicle_id": {
+                "$id": "#/properties/data/properties/status_changes/items/properties/vehicle_id",
+                "type": "string",
+                "description": "The Vehicle Identification Number visible on the vehicle itself",
+                "default": "",
+                "examples": [
+                  "ABC123"
+                ],
+                "pattern": "^(.*)$"
+              },
+              "vehicle_type": {
+                "$id": "#/properties/data/properties/status_changes/items/properties/vehicle_type",
+                "$ref": "#/definitions/vehicle_type",
+                "description": "The type of vehicle"
+              },
+              "propulsion_type": {
+                "$id": "#/properties/data/properties/status_changes/items/properties/propulsion_type",
+                "description": "The type of propulsion; allows multiple values",
+                "$ref": "#/definitions/propulsion_type"
+              },
+              "event_type": {
+                "$id": "#/properties/data/properties/status_changes/items/properties/event_type",
+                "type": "string",
+                "description": "The type of the event",
+                "enum": [
+                  "available",
+                  "reserved",
+                  "unavailable",
+                  "removed"
+                ]
+              },
+              "event_type_reason": {
+                "$id": "#/properties/data/properties/status_changes/items/properties/event_type_reason",
+                "type": "string",
+                "description": "The reason for the event",
+                "enum": [
+                  "service_start",
+                  "user_drop_off",
+                  "rebalance_drop_off",
+                  "maintenance_drop_off",
+                  "user_pick_up",
+                  "maintenance",
+                  "low_battery",
+                  "service_end",
+                  "rebalance_pick_up",
+                  "maintenance_pick_up"
+                ]
```

Comment:
Do we need to do anything to link `event_type` and `event_type_reason` 

ie, you can't have a `removed` event with `user_drop_off` as reason.

Reply:
We [talked about that](https://github.com/CityofSantaMonica/mobility-data-specification/pull/1#discussion_r218934410) over in the original PR but decided to hold off for now. It *may* be possible but it is complex and we need to think on it more.

Reply:
ok.